src/latex/LaTeXWriter.jl

```diff
@@ -87,7 +87,14 @@ _hash(x) = string(hash(x))
 const STYLE = joinpath(dirname(@__FILE__), "..", "..", "assets", "latex", "documenter.sty")
 const DEFAULT_PREAMBLE_PATH = joinpath(dirname(@__FILE__), "..", "..", "assets", "latex", "preamble.tex")
 
-hastex() = (try; success(`latexmk -version`); catch; false; end)
+hastex() = (
+    try
+        success(`latexmk -version`) && return "latexmk"
+        success(`texify --version`) && return "texify"
+    catch
+        return ""
+    end
+)
```

Comment:
Cleaner if it's using just the standard function syntax:

```suggestion
function hastex()
    try
        success(`latexmk -version`) && return "latexmk"
        success(`texify --version`) && return "texify"
    catch
        return ""
    end
end
```